### ISSUE TITLE
Hyprland crashes very often, especially closing windows seems dangerous

### ISSUE THREAD

**Issue by talpadk** [hyprwm/Hyprland#5410]
### Hyprland Version

Hyprland, built from branch main at commit d88d5898807bf6f2462cecfb977b7624dafbf52e dirty (swipe: add events).
Date: Wed Apr 3 20:20:47 2024
Tag: v0.38.0-17-gd88d5898

flags: (if any)

System Information:
System name: Linux
Node name: poseidon
Release: 6.7.9-amd64
Version: #1 SMP PREEMPT_DYNAMIC Debian 6.7.9-2 (2024-03-13)

GPU information: 
06:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Renoir [Radeon RX Vega 6 (Ryzen 4000/5000 Mobile Series)] [1002:1636] (rev d1) (prog-if 00 [VGA controller])

os-release: PRETTY_NAME="Debian GNU/Linux trixie/sid"
NAME="Debian GNU/Linux"
VERSION_CODENAME=trixie
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

plugins:

### Bug or Regression?

Bug

### Description

Crashes when closing a foot terminal (and other windows as well)

### How to reproduce

Basically close any window and there is a high chance of hyprland crashing
Heck sometimes even opening windows crashes hyprland

### Crash reports, logs, images, videos

[hyprlandCrashReport64009.txt](https://github.com/hyprwm/Hyprland/files/14857498/hyprlandCrashReport64009.txt)

**Comment by talpadk** [comment]
My build was build from a pull without  --recurse-submodules 

Rebuilding and testing if it fixes it

**Comment by Agent00Ming** [comment]
#5402 ?

**Comment by vaxerski** [comment]
ye

**Comment by talpadk** [comment]
Don't think so the rebuild allows me to spam delete foot terminals without chrashing, but it did die starting up firefox

**Comment by vaxerski** [comment]
didn't we tell you it's a dupe of 5402 above
